libs/akita/src/lib/store.ts

```diff
@@ -45,6 +46,7 @@ interface StoreSnapshot<S> {
  *   }
  * }
  */
+@Injectable()
```

Comment:
Akita isn't coupled to Angular.

Reply:
I wonder the implications of this: https://angular.io/guide/migration-injectable